### DISCUSSION TITLE
Missing script dev in package json

### DISCUSSION THREAD

**Discussion by choitim1** [open-tutorials/cypress#83] [Q&A]
Вроде все повторял по туториалу.
В чем косяк не могу понять.

![missing](https://user-images.githubusercontent.com/90233385/226784973-a2776416-3f73-4336-9c75-a1e0f7a21c0d.png)

**Comment by breslavsky** [comment]
Из скриншота видно, что терминал не открыт в папке текущего проекта.

А так же, никогда не называйте файлы и папки для разработки с пробелами!

**Comment by choitim1** [comment]
### после установки 
```bash
npm install concurrently
```
 открылся ***localhost***
![concurrently](https://user-images.githubusercontent.com/90233385/227095946-f8cea2ca-74fb-4a2f-974d-6dc473074c12.png)
